airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -516,6 +509,16 @@ def skip_backward_compatibility_tests_fixture(
 
         if previous_connector_version == inputs.backward_compatibility_tests_config.disable_for_version:
             pytest.skip(f"Backward compatibility tests are disabled for version {previous_connector_version}.")
+
+        return False
+
+    @pytest.fixture(name="skip_backward_catalog_tests")
+    def skip_backward_catalog_fixture(
```

Comment:
I'd suggest giving this routine a more meaningful name, otherwise looks good to me. But let's also wait for operations team to review